Packs/Dig/Scripts/Dig/Dig.py

```diff
@@ -0,0 +1,141 @@
+import re
+import subprocess
+import traceback
+from typing import Any, Dict
+
+import demistomock as demisto  # noqa: F401
+from CommonServerPython import *  # noqa: F401
+
+''' STANDALONE FUNCTION '''
+
+
+# Run Dig command on the server and get A record for the specified host
+def dig_result(server: str, name: str):
+    try:
+        if server:
+            server = f"@{server}"
+            dig_output = subprocess.check_output(
+                ['dig', server, name, '+short', '+identify'], stderr=subprocess.STDOUT, universal_newlines=True
+            )
+
+            if not dig_output:
+                raise ValueError("Couldn't find A record for:\n" + name)
+
+            resolved_addresses, dns_server = regex_result(dig_output, reverse_lookup=False)
+
+            return {"name": name, "resolvedaddresses": resolved_addresses, "nameserver": dns_server}
+
+        else:
+            dig_output = subprocess.check_output(
+                ['dig', name, '+short', '+identify'], stderr=subprocess.STDOUT, universal_newlines=True
+            )
+
+            if not dig_output:
+                raise ValueError("Couldn't find A record for:\n" + name)
+
+            resolved_addresses, dns_server = regex_result(dig_output, reverse_lookup=False)
+
+            return {"name": name, "resolvedaddresses": resolved_addresses, "nameserver": dns_server}
+
+    except Exception as e:
+        if isinstance(e, subprocess.CalledProcessError):
+            msg = e.output  # pylint: disable=no-member
+        else:
+            msg = str(e)
+        return_error(msg)
+
+
+# Run Dig command on the server and get PTR record for the specified IP
+def reverse_dig_result(server: str, name: str):
+    try:
+        if server:
+            server = f"@{server}"
+            dig_output = subprocess.check_output(
+                ['dig', server, '+answer', '-x', name, '+short', '+identify'], stderr=subprocess.STDOUT, universal_newlines=True
+            )
+
+            if not dig_output:
+                raise ValueError("Couldn't find PTR record for:\n" + name)
+
+            resolved_addresses, dns_server = regex_result(dig_output, reverse_lookup=True)
+
+            return {"name": name, "resolveddomain": resolved_addresses, "nameserver": dns_server}
+
+        else:
+            dig_output = subprocess.check_output(
+                ['dig', '+answer', '-x', name, '+short', '+identify'], stderr=subprocess.STDOUT, universal_newlines=True
+            )
+
+            if not dig_output:
+                raise ValueError("Couldn't find PTR record for:\n" + name)
+
+            resolved_addresses, dns_server = regex_result(dig_output, reverse_lookup=True)
+
+            return {"name": name, "resolveddomain": resolved_addresses, "nameserver": dns_server}
+
+    except Exception as e:
+        if isinstance(e, subprocess.CalledProcessError):
+            msg = e.output  # pylint: disable=no-member
+        else:
+            msg = str(e)
+        return_error(msg)
```

Comment:
other exceptions will be handled by the try-except in the main function.
```suggestion
    except subprocess.CalledProcessError as e:
        return_error(e.output)
```